packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -158,9 +158,11 @@ class PlanarFreehandROITool extends AnnotationTool {
         // be infinite as the lines become very computationally expensive to draw.
         subPixelResolution: 4,
         interpolation: {
+          editEnabled: false, // used for edit only
           enabled: false,
-          minKnotDistance: 20,
-          editMinKnotDistance: 10, // used for edit only
+          knotsRatioPct: 40,
+          editKnotsRatioPct: 40,
+          forceAllSegment: false,
```

Comment:
Not even sure what this is doing, force All segment to do what? IMO the variable name (since it is in config of the tool) should somehow tells what it does

Reply:
removed it